subprojects/test-runner/service/src/main/kotlin/com/avito/runner/service/worker/DeviceWorker.kt

```diff
@@ -197,6 +201,32 @@ internal class DeviceWorker(
         )
     }
 
+    private suspend fun onDeviceDieWhenExecutingTest(
+        intention: Intention,
+        reason: Throwable
+    ) {
+        deviceListener.onDeviceDied(
+            device = device,
+            message = "DeviceWorker died when executed intention: $intention",
+            reason = reason
+        )
+        deviceListener.onTestCompleted(
```

Comment:
Does it mean that in case of emulator issues test won't retry?

Reply:
Yes this attempt will be LOST